### DISCUSSION TITLE
my elast alert is not sending mail alert

### DISCUSSION THREAD

**Discussion by sandraimmaculate** [Security-Onion-Solutions/securityonion#12679] [Unsupported Versions]
hi, i have installed security onion 2.4 in rocky Linux i have created a play to get an alert when sudo command is executed and yes I'm getting the alert in alert dashboard whenever the sudo command got executed.
I have configured post-fix to send mail from the system i f manually test like:
sendmail example@gmail.com
From: root@gmail.com
Subject: Test mail
Testing Email
.
i'm getting the mail 

and i have configured elastalert by creating my-alert.yml in /opt/so/rules/elastalert/ path which contains contains:

alert:
- "email"
email:
- "youremail@gmail.com"
smtp_host: "smtp.gmail.com"
smtp_port: 465
smtp_ssl: true
from_addr: "youremail@gmail.com"
smtp_auth_file: '/opt/elastalert/rules/smtp_auth_file.txt'

and created /opt/elastalert/rules/smtp_auth_file.txt file also which contains the user name and password and restarted the elastalert 
after that sudo command got executed i got alert in alert dashboard but not in email do i have to do anything else a part from this.

**Comment by InfosecGoon** [comment]
Just to be clear, are you adding this email alert syntax to an Elastalert rule or is it in a file by itself?

**Reply by sandraimmaculate** [reply]
thanks for the reply!

its a playbook alert i have created  a playbook after making my playbook active it has created a rule in elastalert directory 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/f3ffc5c1-8abd-421e-aa97-349003a37e2a)
this is the rule which it has created after making it active 
so do i have to add anything inside this file?
i just followed
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/140578526/07992492-8847-49c4-b0a0-0be1219190f7)

**Reply by sandraimmaculate** [reply]
please suggest!

**Comment by mohasmr** [comment]
hello ,did you find a solution yet?
